psycopg/psycopg/waiting.py

```diff
@@ -232,11 +232,12 @@ def wait_epoll(
             while not fileevs:
                 fileevs = epoll.poll(timeout)
             ev = fileevs[0][1]
+            ready = 0
             if ev & ~select.EPOLLOUT:
-                s = Ready.R
-            else:
-                s = Ready.W
-            s = gen.send(s)
+                ready |= Ready.R
```

Comment:
Here we can drop the `|`

Reply:
I copied this from the implementation of [EpollSelector.select()](https://github.com/python/cpython/blob/3.10/Lib/selectors.py#L473) because I believe it's good to follow this reference.
But yes, the `|` is not needed. Micro-optimization?

Reply:
Yes, micro-optimization: this is pretty much the tightest loop. I'm sure there's more to squeeze here but this was visible.